qlty-check/src/planner/plugin_workspace_entry_finder_builder.rs

```diff
@@ -20,61 +20,47 @@ pub struct PluginWorkspaceEntryFinderBuilder {
     pub paths: Vec<PathBuf>,
     pub file_types: HashMap<String, FileType>,
     pub ignores: Vec<Ignore>,
-    pub cached_git_diff: Option<GitDiff>,
+    pub git_diff: Option<GitDiff>,
     pub prefix: Option<String>,
```

Comment:
I think this `prefix` attribute on `PluginWorkspaceEntryFinderBuilder` struct is no longer needed?